spec/models/finder_schema_spec.rb

```diff
@@ -25,16 +25,17 @@
     end
   end
 
-  describe "#base_path" do
```

Comment:
I decided to drop a good number of these tests as now that we are using Active Model attributes it felt like we were testing Rails.